config/environment.py

```diff
@@ -101,7 +101,7 @@
     See https://github.com/webcompat/css-fixme/
     for more details."""
 
-IS_BLACKLISTED_DOMAIN = ('Anonymous reporting for domain {0} '
+IS_BLOCKED_DOMAIN = ('Anonymous reporting for domain {0} '
                          'is temporarily disabled. Please contact '
                          'miket@mozilla.com '
                          'for more details.')
```

Comment:
I missed this too. it's why the tests are failing.
```suggestion
IS_BLOCKED_DOMAIN = ('Anonymous reporting for domain {0} '
                     'is temporarily disabled. Please contact '
                     'miket@mozilla.com '
                     'for more details.')
```